test/membrane_rtc_engine/hls_endpoint_test.exs

```diff
@@ -166,15 +167,70 @@ defmodule Membrane.RTC.HLSEndpointTest do
       assert_receive({:cleanup, _cleanup_function, ^stream_id})
       refute_received({:cleanup, _cleanup_function, ^stream_id})
     end
+
+    test "check if number of headers is reduced to 1 when resolution is not stable", %{
```

Comment:
We write test names in such a way that they can be read as `test if number of headers is reduced...`

```suggestion
    test "number of headers is reduced to 1 when resolution is not stable", %{
```

Reply:
Fixed.